builds/check_zproject/ci_build.sh

```diff
@@ -10,7 +10,7 @@ set -ex
 # such checkouts are directly in the same parent directory as "this" project.
 cd "$REPO_DIR/.."
 git clone --quiet --depth 1 https://github.com/42ity/libcidr.git cidr
-git clone --quiet --depth 1 -b stable https://github.com/42ity/libsodium.git libsodium
```

Comment:
@jimklimov trying to catch your attention on this (stable Vs master), since there's a diff. that should make it once libsodium is also branched on 2.1.0, but the base libsodium (stable Vs master) is to be clarified